bin/webpack.js

```diff
@@ -1,19 +1,59 @@
 #!/usr/bin/env node
 
+const {
```

Comment:
can we use this similar to other places `const cp = require("child_process")` and use `cp.exec(...)`